pkg/controller/machine/machine_controller.go

```diff
@@ -18,18 +18,25 @@ package machine
 
 import (
 	"context"
+	"fmt"
 	"path"
 	"sync"
+	"time"
 
+	"github.com/go-log/log/info"
+	kubedrain "github.com/openshift/kubernetes-drain"
```

Comment:
This functionality is offered as a library in k/k https://github.com/kubernetes/kubernetes/pull/72827

/cc @errordeveloper